build.gradle

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2021 Aiven Oy
+ * Copyright 2022 Meroxa, Inc.
```

Comment:
We recently updated the copyright rules on our project (if you rebase you see this would make the build fail).
With our current ruling the copyright header would look like the following:
> Copyright 2021 Aiven Oy and jdbc-connector-for-apache-kafka contributors

Would that work for you?

Reply:
Yup, that will work. 👍